upstream-sync.sh

```diff
@@ -36,15 +38,16 @@ fi
 
 echo "::set-output name=has_new_commits::true"
 # display commits since last sync
-echo 'New commits being pulled:' 1>&1
-git log upstream/"${INPUT_UPSTREAM_BRANCH}" "${LOCAL_COMMIT_HASH}"..HEAD --pretty=oneline
+echo 'New commits being synced:' 1>&1
+git log upstream/"${INPUT_UPSTREAM_BRANCH}" "${LOCAL_COMMIT_HASH}"..HEAD ${INPUT_GIT_LOG_FORMAT_ARGS}
 
-# pull from upstream to target_branch
-echo 'Pulling...' 1>&1
-git pull upstream "${INPUT_UPSTREAM_BRANCH}"
-echo 'Pull successful' 1>&1
+# sync from upstream to target_branch
+echo 'Syncing...' 1>&1
+# sync_command examples: "pull", "merge --ff-only", "reset --hard"
+git ${INPUT_GIT_SYNC_COMMAND} "${INPUT_UPSTREAM_BRANCH}"
```

Comment:
Oops, you removed 'upstream' here. Please re-insert that between the expanded vars so the git command works with the expected remote location.

Reply:
good catch